pygame-blob/roundtable.md

```diff
@@ -0,0 +1,43 @@
+# Pyblobs
+
+Today's roundtable will be focused on reviewing the Pyblobs tutorial.
+
+Please [submit a question](insert link here) the day before the roundtable discussion group.
+
+For this discussion, think of your instructor as a mediator and guide, rather than as a lecturer. You are encouraged to do the following:
+    - Whenever possible, you should follow the Ada Online Communication Norms 
+    - You should pose questions to the group, not the instructor.
+    - Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+    - Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try.)
+    - When discussing code, you to use screen sharing through Zoom.
+    - Take notes!
+
+---
+
+### Let's start popcorn style with:
+* What is your favorite snack?
+* Show off Pyblobs 
```

Comment:
As I mentioned above, they won't have there blobs to show off until the next day.